mkt/extensions/models.py

```diff
@@ -62,8 +65,8 @@ def clean_slug(self):
     @property
     def download_url(self):
         return absolutify(reverse(
-            'extension.download',
-            kwargs={'uuid': self.uuid, 'filename': self.filename}))
+            'extension.download_signed',
+            kwargs={'uuid': self.uuid, 'name': self.filename}))
```

Comment:
Maybe we should use `filename`, since there is a `name` field on the add-on itself.